lib/jsdom/living/xmlhttprequest.js

```diff
@@ -590,6 +594,30 @@ module.exports = function createXMLHttpRequest(window) {
     DONE: 4
   });
 
+  function parseContentType(contentType) {
+    const result = { type: "", params: { } };
+
+    if (contentType) {
+      const parts = contentType.split(";");
+      result.type = parts[0];
+      parts.slice(1).forEach(part => {
+        const kvp = part.split("=");
+        result.params[kvp[0].trim()] = kvp[1].trim();
```

Comment:
Needs to handle cases where people to invalid things like `Content-Type: text/html; charset; foo; baz=bar=qux`.